alma/alma-installments-prestashop
# [insurance] Handle insurance with different language

### Reason for change

[Linear task](https://linear.app/almapay/issue/MPP-1049/test-insurance-with-different-language)

### Code changes

Handle Translation for Insurance

### How to test

Install our module.
Configure your Backoffice in French.
Activate Insurance option.
See if the product and attribute are in french

### Checklist for authors and reviewers




### Non applicable
